[MRG] ENH: plot_input() accepts flexible spike_types argument (#157)
jonescompneurolab/hnn-core: doc/whats_new.rst

- Modify plot_dipole() to accept both lists and individual instances of Dipole object, by `Nick Tolley`_ in `#145 <https://github.com/jonescompneurolab/hnn-core/pull/145>`_

> - Update plot_hist_input() to plot_hist() with a flexibly defined spike_types argument, by `Nick Tolley`_ in `#157 <https://github.com/jonescompneurolab/hnn-core/pull/157>`_

Reviewer: I think this should go into the API section below

Reviewer: actually never mind, it's fine here. You also added a new feature. Perhaps modify this to reflect that:

```suggestion
- Update plot_hist_input() to plot_spikes_hist() which can plot histogram of spikes for any cell type, by `Nick Tolley`_ in `#157 <https://github.com/jonescompneurolab/hnn-core/pull/157>`_
```
